fast-cart/src/components/Loading.js

```diff
@@ -0,0 +1,7 @@
+const Loading = () => {
```

Comment:
You can read about React.memo and think about how you could leverage in your components